doc/files.md

```diff
@@ -820,30 +807,39 @@ BoxFile file = new BoxFile(api, "id");
 file.createMetadata(new Metadata().add("/foo", "bar"));
 ```
 
-Update a files Metadata by calling [`updateMetadata(Metadata properties)`][update-metadata].
+Note: This method will only succeed if the provided metadata template is not currently applied to the file, otherwise
+it will fail with a Conflict error. To get to know how to edit existing metadata please
+go to [set metadata](#set-metadata) and [update metadata](#update-metadata) sections.
+
+[create-metadata]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxFile.html#createMetadata-com.box.sdk.Metadata-
+[create-metadata-2]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxFile.html#createMetadata-java.lang.String-com.box.sdk.Metadata-
+[create-metadata-3]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxFile.html#createMetadata-java.lang.String-java.lang.String-com.box.sdk.Metadata-
+[update-metadata]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxFile.html#updateMetadata-com.box.sdk.Metadata-
+
+Set Metadata
+------------
+
+To set metadata on a file, call [`setMetadata(String templateName, String scope, Metadata metadata)`][set-metadata].
+This method will try to create provided metadata on a file. However, if metadata has already been applied to this file,
+it will overwrite values of metadata keys specified in the `metadata` parameter.
```

Comment:
Should we say that it will execute 2 API calls?
Should we say even clearer that if you do not change metadata fields they will remain unchanged? It says it will change values specified but some users were confused.

Reply:
`Should we say that it will execute 2 API calls?`
I think no because sometimes this method can also execute only 1 call.

Reply:
`Should we say even clearer that if you do not change metadata fields they will remain unchanged?`
I added that part